shedlock-provider-hazelcast/src/main/java/net/javacrumbs/shedlock/provider/hazelcast/HazelcastLock.java

```diff
@@ -0,0 +1,99 @@
+package net.javacrumbs.shedlock.provider.hazelcast;
+
+import net.javacrumbs.shedlock.core.LockConfiguration;
+
+import java.io.Serializable;
+import java.time.Instant;
+
+/**
+ * Hazelcast lock entity.
+ * <p>
+ * It's used to persist lock informations into Hazelcast instances (cluster).
+ */
+class HazelcastLock implements Serializable {
+
+    private final String name;
+
+    private final Instant lockAtMostUntil;
+
+    private final Instant lockAtLeastUntil;
+
+    /**
+     * Unique ID of Hazelcast cluster member.
+     * This information is stored to identify the Lock owner and automatically unlock if this member shuddown.
+     */
+    private final String clusterMemberUuid;
+
+    /**
+     * Moment when the lock is expired, so unlockable.
+     * The first value of this is {@link #lockAtMostUntil}.
+     */
+    private Instant unlockTime;
```

Comment:
It's a pity we have this one mutable field here. But I cant figure out how to get rid of it.